x-pack/plugins/canvas/public/components/workpad_header/share_menu/utils.ts

```diff
@@ -47,7 +48,7 @@ export function getPdfJobParams(
   return {
     layout: {
       dimensions: { width, height },
-      id: 'canvas',
+      id: 'canvas' as LayoutTypes, // uses the const value instead of `LayoutTypes.CANVAS`, to avoid the screenshotting plugin being a required bundle with canvas
```

Comment:
It would be easier use `LayoutTypes.CANVAS` instead of the hardcoded `'canvas'` string, but that would require "screenshotting" to be a required bundle in Canvas' kibana.json file. That is why there is `as LayoutTypes` on this line. Hopefully the inline comment explains that.

Reply:
Thanks for this suggestion. It looks like we wanted to use the enum so we have a field to use instead of a string literal, but with a string union type it's the same thing. I am going with your suggestion, because it means having many fewer `import` statements :+1: